docs/source/build_network.rst

```diff
@@ -1,1466 +1,17 @@
 Building Your First Network
 ===========================
 
-.. note:: The Build your first network (BYFN) tutorial has been deprecated. If
+.. note:: The Build your first network (BYFN) tutorial has been removed. If
           you are getting started with Hyperledger Fabric and would like to deploy
           a basic network, see :doc:`test_network`. If you are deploying Fabric
           in production, see the guide for :doc:`deployment_guide_overview`.
 
-The build your first network (BYFN) scenario provisions a sample Hyperledger
-Fabric network consisting of two organizations, each maintaining two peer
-nodes. It also will deploy a Raft ordering service by default.
+          You can still find the BYFN tutorial in earlier Fabric documentation versions
+          and a link is provided below to BYFN in Fabric 2.1 for your convenience.
+          https://hyperledger-fabric.readthedocs.io/en/release-2.1/build_network.html
```

Comment:
I think we can link directly using this format. Maybe something like:

You can still find the BYFN tutorial in previous Fabric documentation versions:
- `2.1 <https://hyperledger-fabric.readthedocs.io/en/release-2.1/build_network.html>`_
- `2.0 <https://hyperledger-fabric.readthedocs.io/en/release-2.0/build_network.html>`_
- `1.4 <https://hyperledger-fabric.readthedocs.io/en/release-1.4/build_network.html>`_

Reply:
I do not know if we need to link to 2.0 since 2.1 is just 2.1 plus bug fixes.

Reply:
yeah, linking to v1.4 and v2.1 would be sufficient, always good to direct people to the latest versions per release.

Reply:
Agree Dave, saw Pam's note before yours.  Will fix.